torch/csrc/jit/passes/utils/check_alias_annotation.cpp

```diff
@@ -25,7 +25,7 @@ IValue deepCopy(const IValue& self) {
 
   // Lists of ivalues should recursively deep copy their contents
   if (self.isGenericList()) {
-    c10::List<IValue> newList;
+    auto newList = c10::impl::GenericList(c10::impl::deprecatedUntypedList());
```

Comment:
we can probably copy self's type here

Reply:
same here, I'll fix these call sites separately.